src/app/jimm-facade.js

```diff
@@ -1,7 +1,4 @@
-const {
-  autoBind,
-  createAsyncHandler,
-} = require("@canonical/jujulib/api/transform.js");
+const { autoBind } = require("@canonical/jujulib/dist/api/utils.js");
```

Comment:
The jimm facade is the only one still manually generated. This is likely changing in the future but for now....here it is.